composer.json

```diff
@@ -2,10 +2,10 @@
     "name": "somethingdigital/invalidate-admin-passwords",
     "description": "Invalidate all admin passwords",
     "require": {
-        "php": ">=7.0 <7.5"
+        "php": "^7.4 || ~8.1.0"
```

Comment:
Can we still keep the old php requirements but add in 8.1?

Reply:
Changed